packages/task/src/common/task-protocol.ts

```diff
@@ -97,6 +97,9 @@ export interface TaskExitedEvent {
     // Exactly one of code and signal will be set.
     readonly code?: number;
     readonly signal?: string;
+
+    readonly label: string;
```

Comment:
@vzhukovskii shouldn't the new `label` and `processType` properties be optional?

Reply:
Don't think so, in TaskConfiguration these fields are mandatory.

Reply:
I was just pointing out that making them mandatory means the update to the `TaskExitedEvent` is a breaking change.
Perhaps we can instead keep an optional `config: TaskConfiguration` property?

Reply:
Updated PR, now there is only optional config field.

Reply:
Thanks, do you mind squashing your commits into a single commit?

Reply:
Sure, I'm going to squash all changes into a single commit before merge. Just waiting for other feedbacks.